app/controllers/admin/snap_applications_controller.rb

```diff
@@ -17,5 +17,23 @@ class SnapApplicationsController < Admin::ApplicationController
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
+    #
+    def resend_fax
+      application = SnapApplication.find(params[:id])
+      application.exports.faxed.succeeded.update_all(status: :failed, metadata:
+                                                     "Reset by admin")
+      Export.create_and_enqueue!(snap_application: application, destination:
+                                 :fax)
+      flash[:notice] = "Resent fax #{application.email}!"
```

Comment:
a bit confused about why we'd put the email address in the flash if we are sending a fax?

Reply:
I was trying to figure out what data would be most useful, and totally blanked on using signature + fax recipient number.